homework/index.js

```diff
@@ -29,12 +31,115 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function addCont(login, img, contributeNumber, parent, link) {
```

Comment:
What do you mean with `addCont`? Is it `addContributor`? How much effort is it to type those few more characters to take away any doubt on what you mean?

You are passing a log of parameters to this function. Except for `parent` they are all properties of a contributor object. Therefore this function would look less complicated and easier to understand if you just pass two parameters:

```js
function addContributor(contributor, parent) {
  ...
}
```